lib/xml/create_shipment.xml

```diff
@@ -43,8 +43,10 @@
         </v2:electronicAddress>
       </v2:recipientContact>
       <v2:recipientAddress>
+        <buildingName><%= try(:building_name) %></buildingName>
```

Comment:
Should we render `<buildingName/>` only when `buildingName` is available?

Reply:
I don't see why - it's ok to have empty tags in xml and other fields have this already